back/node_watcher/src/util/statuses.ts

```diff
@@ -30,3 +30,8 @@ export const motionStatus = {
   PROPOSED: 'Proposed',
   VOTED: 'Voted',
 };
+
+export const treasuryProposalStatus = {
+  PROPOSED: 'Proposed',
+  TABLED: 'Tabled',
```

Comment:
`Tabled` isn't a treasury event. What you'll care about for the status task is `Awarded` and `Rejected`.

Reply:
done